apps/docs/src/app/platform/component-docs/platform-button/platform-button-docs.component.html

```diff
@@ -1,8 +1,7 @@
 <fd-docs-section-title [id]="'Button-types'" [componentName]="'button'"> Button Types </fd-docs-section-title>
 <description>
-    The default button does not require modifier. Use
-    <code class="code-snippet">[type]="'standard'", [type]="'positive'", [type]="'medium'"</code>, or
-    <code class="code-snippet">[type]="'negative'"</code> to modify the type of the button.
+    The button component follows fundamental-styles for button guidelines. The default button type is "Default button",
```

Comment:
I would not copy the documentation from the core package

Reply:
> The documentation is not generated properly - the name of the component is all lower cases
> <img alt="Screen Shot 2020-06-20 at 2 19 41 PM" width="296" src="https://user-images.githubusercontent.com/4380815/85208793-27a9cc80-b301-11ea-8b85-53ba92518371.png">

Updated
<img width="394" alt="Screen Shot 2020-06-22 at 12 05 26 PM" src="https://user-images.githubusercontent.com/53487468/85255844-b1cc6f00-b480-11ea-8ee1-76e474bebaae.png">

Reply:
<img width="852" alt="Screen Shot 2020-06-22 at 12 18 45 PM" src="https://user-images.githubusercontent.com/53487468/85256919-9498a000-b482-11ea-8d7d-a2ebaf7cafea.png">
Updated